README.md

```diff
@@ -9,16 +9,22 @@ When working with this logger library, please make sure you use one of the init
 Note: You can still create your own standard logger by using the `NewLogger` function (check [Logrus](https://github.com/sirupsen/logrus/blob/master/logger.go#L69) for more details).
 
 
-### Creating an Entry
-The library maintains most of the Logrus's way of logging, but it adds some default fields (`@time`, `transaction_id` and `service_name`) for the created logging entries.
-For entry creation you can use the following methods:
-- `NewEntry` - with `transaction_id` as a parameter
-- `NewMonitoringEntry` - with `transaction_id`, `eventName` and `contentType` as parameters. Beside these, a `monitoring_event=true` field will also be added to the entry. This message will be picked up by the monitoring services and dashboards.
+### Logging a Monitoring Event
+The library maintains most of the Logrus's way of logging, but it adds some default fields 
```

Comment:
Minor grammar tweak - "The library is `logrus` compatible, but it includes a few default fields which help facilitate the monitoring of key application events."